File: Week 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n ;
    do
    {
        n = get_int("height=");
    }
    while (n <= 0 || n > 8);
    // for printing columns     
    for (int i = 0 ; i < n ; i++)
    {   
        // for printing spaces
        for (int j = n - 1 ; j > i ; j--)
        {
            printf(" ");
        }   

        // for printing hashes
        for (int k = 0 ; k <= i ; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}
       
   

File: Problem Set 6 - Python/C to Python Sentimental/mario/less/mario.py
from cs50 import get_int

n = get_int("height=")

while n <= 0 or n > 8:
    n = get_int("height=")
    
for i in range(n):
    for j in range(n):
        if (i + j) < (n - 1):
            print(" ",end="")
        else:
            print("#", end="")
    print()    
            